ComponentKit/Components/CKImageComponent.h

```diff
@@ -18,14 +18,16 @@ struct CKComponentSize;
 @interface CKImageComponent : CKComponent
 
 /**
- Uses a static layout with the image's size.
+ Uses a static layout with the image's size and apply additional attributes.
  */
-+ (instancetype)newWithImage:(UIImage *)image;
++ (instancetype)newWithImage:(UIImage *)image
+                  attributes:(const CKViewComponentAttributeValueMap &)attributes;
 
 /**
- Uses a static layout with the given image size.
+ Uses a static layout with the given image size and applies additional attributes.
  */
 + (instancetype)newWithImage:(UIImage *)image
+                  attributes:(const CKViewComponentAttributeValueMap &)attributes
                         size:(const CKComponentSize &)size;
 
 @end
```

Comment:
This is so sad that we have multiple initializers with one of our core components.